templates/table.js.tera

```diff
@@ -559,6 +560,11 @@ for (o of config.ellipsis) {
 {% for row, heatmap in header_heatmaps %}
     colorizeHeaderRow{{ loop.index0 }}();
 {% endfor %}
+
+for (o of config.header_ellipsis) {
+    shortenHeaderRow(o.index, o.ellipsis, {% if not header_labels | length == 0 %}true{% else %}false{% endif %});
```

Comment:
Can't you just say `{{ header_labels | length == 0 }}` here?

Reply:
Yep just blindly copied it from the earlier code.

Reply:
It would need to be negated though.